chartiq/development/js/chartiq.js

```diff
@@ -11452,7 +11452,7 @@ CIQ.ChartEngine.prototype.dateFromTick = function (
 	}
 
 	if (nativeDate) {
-		result = new Date(dt.getTime());
+		result = new Date(dt?.getTime());
```

Comment:
can we find why `dt` is undefined? may be we are not passing it?
`new Date(undefined)` will be an Invalid date.

Reply:
I have done it in another PR: https://github.com/binary-com/SmartCharts/pull/1199 😊